DCC.combined-schema.json

```diff
@@ -3,21 +3,42 @@
   "$id": "https://id.uvci.eu/DCC.combined-schema.json",
   "title": "EU DCC",
   "description": "EU Digital Covid Certificate",
-  "$comment": "Schema version 1.2.1",
-  "required": [
-    "ver",
-    "nam",
-    "dob"
-  ],
+  "$comment": "Schema version 1.3.0",
   "type": "object",
+  "oneOf": [
```

Comment:
Is this really correct? Will generate funny code.

Reply:
I thought we said that it should be a business rule to only include one of the types. And have exactly one element.

IMHO this change is too big and people generating code will have to change too much.